content/en/blog/2023/sig-instrumentation-spotlight.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: blog
+title: "Spotlight on SIG Instrumentation"
+slug: sig-instrumentation-spotlight-2023
+date: 2023-01-31
+author: "Imran Noor Mohamed(Delivery Hero)"
+---
+
```

Comment:
Add **Author**: \<you\> in the main body.

Reply:
```
author: "Imran Noor Mohamed(Delivery Hero)"
```
in the front matter is fine - IMO